nltk/tokenize/stanford_segmenter.py

```diff
@@ -70,9 +70,8 @@ def __init__(self,
                 searchpath=(), url=_stanford_url,
                 verbose=verbose)
 
-        # This is passed to java as the -cp option, the segmenter needs slf4j.
-        self._stanford_jar = os.pathsep.join(
-            [_ for _ in [stanford_segmenter, slf4j] if not _ is None])
+        # Update: The new version of stanford-segmenter-2016-10-31 don't need the path of slf4j
+        self._stanford_jar = stanford_segmenter
```

Comment:
No, the previous one is also a string because it's using "join", which is concatenating the two element.